cli/azd/cmd/login.go

```diff
@@ -128,7 +128,7 @@ func newLoginCmd() *cobra.Command {
 		Log in to Azure.
 
 		When run without any arguments, log in interactively using a browser. To log in using a device code, pass
-		--use-device-code. To log in under a particular tenant, pass --tenant-id.
+		--use-device-code.
```

Comment:
I think it's best not to explicitly document this for now, as the concept of `--tenant-id` may be confusing for a user. It really is trying to ensure conditional access policies are met with the AD tenant.